lib/ColorThief/ColorThief.php

```diff
@@ -259,7 +259,7 @@ protected static function isNonWhite($color)
     private static function vboxFromHistogram(array $histo)
     {
         $rgbMin = [PHP_INT_MAX, PHP_INT_MAX, PHP_INT_MAX];
-        $rgbMax = [0, 0, 0];
+        $rgbMax = [-PHP_INT_MAX, -PHP_INT_MAX, -PHP_INT_MAX];
```

Comment:
I think `[0, 0, 0]` is still valid as color components are always positive integers, but anyway, `-PHP_INT_MAX` is fine too.